### PR TITLE
Some bugs fixed on front-end

### PR DESCRIPTION
**Pull request by EldanizAkbar**
- Completed the design for all pages
- Added error handling for creating and updating restaurants
- Added a modal dialogue box for deleting restaurants

### CODE REVIEW

File: grable_frontend/src/views/ARestaurant.vue [hunk 1]
```diff
@@ -1,21 +1,38 @@
 <template>
-  <div class="A Post">
+  <div class="ARestaurant">
```
Comment (smusali) [inline grable_frontend/src/views/ARestaurant.vue:2]: No whitespace?

**Review by smusali — CHANGES_REQUESTED** [review]
@EldanizAkbar can u also please attach the screenshot of the page created using these changes? Thanks

File: grable_frontend/src/views/ARestaurant.vue (hunk 1 above)
Comment (EldanizAkbar) [reply grable_frontend/src/views/ARestaurant.vue:2]: > No whitespace?

I renamed the class of the div to a more logical name.

**Comment by EldanizAkbar** [comment]
> @EldanizAkbar can u also please attach the screenshot of the page created using these changes? Thanks

a restaurant
add restaurant error handling
delete restaurant
home

some screenshots from the project

**Review by smusali — APPROVED** [review]
lgtm

**Pull request merged**